examples/guides/mp-restful-webservice/src/main/docs/index.adoc

```diff
@@ -0,0 +1,487 @@
+
```

Comment:
STYLE: A general observation - Use either the title case or the sentence case, consistently for all the sections.

Reply:
Do you have a strong suggestion one way or the other?

Reply:
The [Oracle Style Guide](http://st-doc.us.oracle.com/dp/1/doctools.101/e23911/grammar.htm#i1005871) prescribes the title case for section headings in docs that we publish on docs.oracle.com. Helidon docs aren't published to docs.oracle.com, but I suggest we follow the same standard.

Reply:
OK. Will do.

Reply:
@kumar-dhanagopal if we should follow any standard it should not be proprietary. Either we find one that is widely used by open source projects or we make one for ourselves.

Reply:
Also please avoid internal links on github.

Reply:
Good point about internal links, Romain... I forgot the context; sorry!

Reply:
About the heading style... I bow to Kumar's greater wisdom on this. Whatever you say, @kumar-dhanagopal, we'll go with!

About internal links: @romain-grecourt Do internal links (internal cross-references, i.e. `<<xxx>>`) not work on GitHub?? There are several scattered throughout the guide. Do we need to change them to  `link:`-style? There are warnings in the AsciiDoctor user's guide against that practice, at least for inter-document cross-references (which we use). 
https://asciidoctor.org/docs/user-manual/#inter-document-cross-references

Reply:
Personally, I like sentence case, because it is less formal. Title style is old school, but its use won't disappear for a while. 

We can't go horribly wrong with either choice. Consistency is key.

I suggest we pick title case as our standard, unless anyone has strong concerns, and use it consistently for content we write going forward.

Reply:
> ... Do internal links (internal cross-references, i.e. `<<xxx>>`) not work on GitHub?

I think @romain-grecourt 's comment was about links to Oracle-internal docs, like the style guide link in my earlier comment.

Reply:
I would vote for sentence case, I think, because of the "new-school-ness" as you pointed out. I don't have _really_ strong feelings about this, but as the "old-school" way grows even older as time goes on our content might seem increasingly old and remain that way as we strive for consistency if we adopt the title case convention now.

Reply:
@kumar-dhanagopal Any reaction to my recent thoughts from yesterday... Adopting "old-school" title case would lock us in going forward and (possibly) make us seem older and older school as time goes on.

Reply:
Sorry, I missed responding to that...

Good point! 

I looked at a few other sites, to get an idea of the direction the "industry" is taking. 

Kubernetes: No real consistency: title case on some pages (https://kubernetes.io/docs/setup/) and sentence case in others (e.g., https://kubernetes.io/docs/setup/independent/install-kubeadm/)

FnProject: Again no consistency: https://github.com/fnproject/docs/blob/master/fn/general/introduction.md 

Docker: Sentence case (https://docs.docker.com)

Apache Kafka: Mostly title case, but stray cases of sentence case usage (http://kafka.apache.org/documentation/)

I don't have time right now to do more extensive research. But if we had to take a call right now, I agree with you with the sentence case would give our content a more conversational and modern look. Let's go with that.

Reply:
Thanks for the research and your thoughts. I'll change the MP guide and check the SE one to adopt sentence case.

Reply:
Perhaps we can publish as is now, and pick up this task later.  I would be happy to check & update the case for all the section titles, Tim, if that's ok by you of course.

Reply:
Ha! I'm already part-way through the MP changes (luckily there are not so many headings affected) and the changes to the SE guide will be similarly quick. There will be plenty of heading fix-up in other documents to tend to later!

Reply:
Done